src/hub/tests/unit/stripe/customer/test_stripe_customer.py

```diff
@@ -9,14 +9,18 @@
 import flask
 import stripe
 import requests
+import connexion
+from flask import g
```

Comment:
separate `from imports` from `imports` in external and internal code

Reply:
@scottidler updated imports, please review.